adage/genes/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class GenesConfig(AppConfig):
+    name = 'genes'
```

Comment:
Yes, `apps.py` is the new mechanism to start a Django app. If you read this section Django doc:
https://docs.djangoproject.com/en/2.2/intro/tutorial02/#activating-models
you can see that the recommended way to add a new app is now:
```
INSTALLED_APPS = [
    ... 
    genes.apps.genesConfig
    ...
```
I didn't write it that way because I don't want to blow your mind. :laughing:

Reply:
Whoa!

😀 💥 💀 

GitHub really needs a :mindblown: emoji 🤯